ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -383,20 +428,52 @@ export default defineComponent({
     }
 
     const getOwnerStatus = (ownerStatus: string): string => {
-      if (ownerStatus) {
+      if (ownerStatus === 'PREVIOUS') {
         if (ownerStatus === 'PREVIOUS') return 'HISTORICAL'
         else return ownerStatus
       } else return ''
     }
 
+    // return adaptive text for owner status count(s)
+    const getOwnerStatusText = (item: ManufacturedHomeSearchResultIF): string => {
+      var returnText = ''
+      if (item.activeCount > 0) {
+        returnText += 'ACTIVE'
+        if (item.activeCount > 1) returnText += ' (' + item.activeCount + ')'
+        hasMultipleStatus(item) ? returnText += ',\n' : returnText += '\n'
+      }
+      if (item.exemptCount > 0) {
+        returnText += 'EXEMPT'
+        if (item.exemptCount > 1) returnText += ' (' + item.exemptCount + ')'
+        hasMultipleStatus(item) ? returnText += ',\n' : returnText += '\n'
+      }
+      if (item.historicalCount > 0) {
+        returnText += 'HISTORICAL'
+        if (item.historicalCount > 1) returnText += ' (' + item.historicalCount + ')'
+      }
+      return returnText
+    }
+
+    const hasMultipleStatus = (item: ManufacturedHomeSearchResultIF): boolean => {
+      return (item.activeCount > 0 && item.exemptCount > 0) ||
+             (item.activeCount > 0 && item.historicalCount > 0) ||
+             (item.exemptCount > 0 && item.historicalCount > 0)
+    }
+
+    const noSelectedOwner = (item: ManufacturedHomeSearchResultIF): boolean => {
+      var filteredResults = localState.results?.filter(result => result.mhrNumber === item.mhrNumber)
+      filteredResults = filteredResults.filter(result => result.selected === true)
```

Comment:
Can be simplified to `!!result.selected` or if this property is already of type boolean, `result.selected` would suffice.